qemu/tests/virtio_net_dpdk.py

```diff
@@ -307,8 +325,7 @@ def launch_test(session, generator1, generator2,
     def start_moongen(generator1, mac, port_id, running_time):
 
         file = '/home/MoonGen/examples/udp-throughput.lua'
-        cmd = "pkill MoonGen ; rm -rf /tmp/throughput.log ; sleep 3"
```

Comment:
Would you help introduce why remove it from here?

Reply:
Since it belong to clear env part,  I add it into function unbind_dpdk_nic